lib/github.rb

```diff
@@ -111,19 +118,33 @@ def self.puts_pull_request(pr, options = {})
       puts prefix.nil? ? s : prefix + s
     end.curry.call(options[:prefix])
 
+    color = STDOUT.isatty
```

Comment:
I find this rather distasteful, referencing a static state inside a method, instead of passing in an arg. But this is how you want it and it works, so I'll do it.